common/src/main/java/com/datastrato/gravitino/config/ConfigEntry.java

```diff
@@ -37,6 +37,7 @@ public class ConfigEntry<T> {
   @Getter private boolean isDeprecated;
 
   private boolean isOptional;
+  private boolean isNoDefault;
```

Comment:
Can we use another name, `isNoDefault` seems a bit awkward.

Reply:
How about `hasNoDefault`?

Reply:
Looks good.

Reply:
Fixed.